### PR TITLE
Fix broken UTs in Windows

### PR DESCRIPTION
**Pull request by v1v**
Fixes broken tests 

Closes https://github.com/jenkinsci/docker/issues/1014

### CODE REVIEW

**Comment by v1v** [comment]
Nearly there,  I fixed half of the tests :)

File: tests/plugins-cli/ref/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
 FROM bats-jenkins-install-plugins
 
-RUN rm -rf /usr/share/jenkins/ref ; jenkins-plugin-cli --plugins junit:1.28 ant:1.3
+RUN rm -rf /usr/share/jenkins/ref ; jenkins-plugin-cli --plugins junit:1.6 ant:1.3 mesos:0.13.0 git:latest filesystem_scm:experimental docker-plugin:1.1.6
```
Comment (v1v) [inline tests/plugins-cli/ref/Dockerfile:3]: For some reason, the BATS were not failing even though the PESTER ones were failing for windows. So I decided to change also the linux docker image too

**Comment by slide** [comment]
I was actually working on this as well...I think I have it working for all the tests including the ones that are commented out.

**Comment by v1v** [comment]
> I was actually working on this as well...I think I have it working for all the tests including the ones that are commented out.

Then please discard this PR. Next time I'll sync up to avoid duplicate efforts. In any case I enjoyed my learning with pester :)

**Pull request closed without merging**